app/global/MainModule.java

```diff
@@ -160,6 +162,17 @@ private Props getHostProviderProps(final HostProviderFactory provider, final Env
         return provider.create(config.getConfig("hostProvider"), ConfigurationHelper.getType(environment, config, "hostProvider.type"));
     }
 
+    @Provides
+    @Singleton
+    @SuppressFBWarnings(value = "UPM_UNCALLED_PRIVATE_METHOD", justification = "Invoked reflectively by Guice")
+    private Mailer getEmailTransport(final Config config) {
+        final String host = config.hasPath("mail.smtp.host") ? config.getString("mail.smtp.host") : "localhost";
+        final Integer port = config.hasPath("mail.smtp.port") ? config.getInt("mail.smtp.port") : 25;
```

Comment:
`mail.smtp.{host,port}` are the properties that simplejavamail usually takes its config from, so I cribbed those names. Am open to change.